tests/unit_tests/data/experience/sub_path/exp_test-exp-1.yaml

```diff
@@ -0,0 +1,793 @@
+exp_id: test-exp-1
+experience_text: 'User Query: show top 3 data in ./demo_data.csv
+
+
+  Best Practice: When the user provides a new file path, update the code to use the
+  new file path and check if the file exists before attempting to read it.
+
+
+  Mistakes to Avoid:
+
+  1. Not checking if the file exists before attempting to read it, which can lead
+  to FileNotFoundError.
+
+  2. Not updating the file path when the user provides a new one.
+
+
+  Critical Information:
+
+  1. Use os.path.exists(file_path) to check if the file exists before attempting to
+  read it.
+
+  2. Update the file path in the code when the user provides a new one.'
+raw_experience_path: D:\TaskWeaver\tests\unit_tests\data\experience\raw_exp_test-exp-1.yaml
+embedding_model: all-mpnet-base-v2
```

Comment:
Is it okay to have absolute path for corss file reference in tests?

Reply:
:( will remove